resources/articles/shared-web-credentials-and-google-smart-lock.md

```diff
@@ -0,0 +1,65 @@
+:title Shared Web Credentials and Google Smart Lock
+:frontpage
+:category api-integration
+:body
+
+Shared Web Credentials for iOS and Google Smart Lock for Android will simplify the login step for all users using native apps.
+User having enabled these features, will be prompted in enabled apps to choose which credentials to use, and they will be automatically filled in.
+
+### Configuration
+
+Shared Web Credentials and Google Smart Lock can be configured in SelfService, under Basic settings.
```

Comment:
Maybe you could link SelfService to [http://localhost:3000/selfservice/access/](http://localhost:3000/selfservice/access/)?